### PR TITLE
Add brand new logo to the project

### PR DESCRIPTION
**Pull request by caneco**
Here's the brand new logo for the FakerPHP package:

socialcard

I hope you like it 😊

### CODE REVIEW

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
No idea what happened but conflicts?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Maintainers
+
+**FakerPHP** logo is designed and maintained by [Caneco](https://twitter.com/caneco).
+
+## License
+
+All rights reserved, but with the following extra conditions:
+
+- It is **OK** to use the FakerPHP logo in the following cases:
+    - In marketing materials for technical events, e.g. meetups, hackathons, conferences and workshops that are related to Laravel.
```
Comment (localheinz) [inline README.md]: Do we need a reference to Laravel here?

Comment (pimjansen) [reply README.md]: Agree, no need to have specifics here

Comment (caneco) [reply README.md]: Oops… really sorry about that. #BadCopyPaste.

Should be changed to:

> … conferences and workshops that are related to **PHP or FakerPHP**.

Do you agree?

**Comment by caneco** [comment]
@localheinz && @pimjansen regarding the artwork please use the `avatar@4x.png` to change globally the FakerPHP avatars.

And also don't forget to change the "Social preview" of the repos with the `socialcard-github.png` file.

**Here:** 👇

image

File: README.md (hunk 1 above)
Comment (pimjansen) [reply README.md]: Agreed

**Comment by pimjansen** [comment]
Thanks @caneco

**Comment by caneco** [comment]
My pleasure 😊

**Comment by caneco** [comment]
@pimjansen @GrahamCampbell can you use the `[avatar@4x.png](https://github.com/FakerPHP/Artwork/blob/main/src/avatar%404x.png)` file for the profile picture of the organisation?

That file will give a better breathing space.

**Comment by GrahamCampbell** [comment]
@caneco Are you OK with us changing the license to CC BY-SA, or would you prefer CC BY-NC-SA. Currently, we have conflicting information where the LICENSE file says MIT, which allows people to sell t-shirts, and the README which says no.

**Comment by caneco** [comment]
@GrahamCampbell it would probably be better to have a **CC BY-NC-SA** license. That will secure the integrity of the organisation brand.

**Pull request merged**